### PR TITLE
Bash ifs replaced by matrix.includes

### PR DESCRIPTION
**Pull request by lavish205**
Is this what exactly we have to do for all ifs?

### CODE REVIEW

**Comment by lavish205** [comment]
#511

**Comment by Natim** [comment]
I can still see ifs like this one here: ``if [[ $ACTION != loadtest_tutorial && $ACTION != loadtest_simulation ]];``

**Comment by lavish205** [comment]
@Natim how to handle two condition like `if [[ $ACTION != loadtest_tutorial && $ACTION != loadtest_simulation ]];` in travis?

**Comment by Natim** [comment]
Why woud you like to do that?

**Comment by leplatrem** [comment]
@lavish205 do you need help on this ?

**Comment by lavish205** [comment]
@leplatrem yes, I am unable to figure out how to fix it

**Comment by leplatrem** [comment]
Me neither to be honest !

Let's close this and let's see if somebody is comfortable with that :)

Thanks for your patience, keep up!

**Comment by Natim** [comment]
I can work on this, @lavish205 can you make sure I have permission to write in your branch?

**Comment by Natim** [comment]
Apparently I have the permission already. Let's see if travis is happy with the changes.

File: .travis.yml [hunk 1]
```diff
@@ -6,29 +6,34 @@ addons:
...
+    - env: ACTION=loadtest_tutorial
+      before_script: echo 'Tutorial'
+      script: make loadtest-check-tutorial
+    - env: ACTION=loadtest_simulation
+      before_script: echo 'Simulation'
+      script: make loadtest-check-simulation
+    - env: TOX_ENV=flake8
+      install: pip install flake8
+      before_script: echo 'flake8'
+      script: flake8 kinto
```
Comment (leplatrem) [inline .travis.yml]: Since we have a tox rule for this, the script might not be necessary

Comment (Natim) [inline .travis.yml]: Yes it was an optimization not to install all the dev dependencies to validate flake8.

**Pull request merged**